classes.js

```diff
@@ -0,0 +1,92 @@
+// classes for tic-tac-math
+export {Tile, Board};
+
+
+class Tile {
+    constructor(index) {
+        this.index = index;
+        this.symbol = '';
+        
+    }   
+    
+    tileUnclaimed() {
```

Comment:
in our diagrams we have a function `squareIsEmpty` in the `Board` class. maybe rename it and move it there?